test/specs/core.specs.js

```diff
@@ -47,8 +47,6 @@ describe('Core specs', function() {
         e.preventDefault()
       })
     })
-    // start router
-    route.start()
```

Comment:
Ah, I gotcha. The problem is `setTimeout`. We need some waits in the test, maybe. I'll try it.